web/tests/compose_ui.test.js

```diff
@@ -279,23 +279,25 @@ run_test("quote_and_reply", ({override, override_rewire}) => {
             textarea_caret_pos = arg;
             return this;
         }
-        /* istanbul ignore if */
-        if (typeof arg !== "string") {
-            console.info(arg);
-            throw new Error("We expected the actual code to pass in a string.");
-        }
+        // Not used right now, but could be in future.
```

Comment:
Since the `quote_and_reply` function now only uses `caret` with either no argument or a number passed in, all this commented out code never runs and has lost test coverage. I wasn't sure if I should remove this, hence have commented it for now